source/Stargate-Model/NeoJSONWriter.extension.st

```diff
@@ -0,0 +1,35 @@
+Extension { #name : #NeoJSONWriter }
+
+{ #category : #'*Stargate-Model' }
+NeoJSONWriter >> mapZincEvent [
+
+	self for: ZnHeaders customDo: [ :mapping | 
+		mapping writer: [ :jsonWriter :headers | 
+			jsonWriter writeMap: headers ] ].
```

Comment:
Some headers will have sensitive data (like the `Authorization` ones), so we need to mask these ones to not expose sensitive information in the logs.